tox.ini

```diff
@@ -1,22 +1,21 @@
 [tox]
 skip_missing_interpreters = True
 skipsdist = True
-usedevelop = True
 
 envlist =
-    py{27,33,34,35}-dj{18,19,110}-wt16
+    py{27,33,34,35}-dj{18,19,110}-wt{18,19,110,111}
```

Comment:
FYI: Wagtailformblock package, for running the tests its always using wagtial 1.11  [Wagtailformblock](https://github.com/LUKKIEN/wagtailformblocks/commit/6effb79a04762bca940de0f478454ec8e6916f5e)